client/state/media/actions.js

```diff
@@ -6,21 +6,78 @@ import { castArray } from 'lodash';
 /**
  * Internal dependencies
  */
-import { MEDIA_DELETE, MEDIA_RECEIVE } from 'state/action-types';
+import {
+	MEDIA_DELETE,
+	MEDIA_RECEIVE,
+	MEDIA_REQUEST,
+	MEDIA_REQUEST_FAILURE,
+	MEDIA_REQUESTING } from 'state/action-types';
```

Comment:
Sure, but what's the guideline for this? https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/javascript.md#objects